src/XPurchase/VirtoCommerce.XPurchase/Schemas/PurchaseSchema.cs

```diff
@@ -44,33 +44,23 @@ public void Build(ISchema schema)
                 Name = "cart",
                 Arguments = new QueryArguments(
                         new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "storeId", Description = "Store Id" },
-                        new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "userId", Description = "User Id" },
+                        new QueryArgument<StringGraphType> { Name = "userId", Description = "User Id" },
                         new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "currencyCode", Description = "Currency code (\"USD\")" },
                         new QueryArgument<StringGraphType> { Name = "cultureName", Description = "Culture name (\"en-Us\")" },
                         new QueryArgument<StringGraphType> { Name = "cartName", Description = "Cart name" },
                         new QueryArgument<StringGraphType> { Name = "type", Description = "Cart type" }),
                 Type = GraphTypeExtenstionHelper.GetActualType<CartType>(),
                 Resolver = new AsyncFieldResolver<object>(async context =>
                 {
-                    //TODO: Move to extension methods
-
-                    var storeId = context.GetArgument<string>("storeId");
-                    var cartName = context.GetArgument("cartName", "default");
-                    var userId = context.GetArgument<string>("userId");
-                    var cultureName = context.GetArgument<string>("cultureName");
-                    var currencyCode = context.GetArgument<string>("currencyCode");
-                    var type = context.GetArgument<string>("type");
-
-                    var getCartQuery = new GetCartQuery(storeId, type, cartName, userId, currencyCode, cultureName);
+                    var getCartQuery = context.GetCartQuery<GetCartQuery>();
                     getCartQuery.IncludeFields = context.SubFields.Values.GetAllNodesPaths().ToArray();
                     var cartAggregate = await _mediator.Send(getCartQuery);
                     if (cartAggregate == null)
                     {
-                        var createCartCommand = new CreateCartCommand(storeId, type, cartName, userId, currencyCode, cultureName);
+                        var createCartCommand = new CreateCartCommand(getCartQuery.StoreId, getCartQuery.CartType, getCartQuery.CartName, getCartQuery.UserId, getCartQuery.CurrencyCode, getCartQuery.CurrencyCode);
```

Comment:
The last parametr is `getCartQuery.CultureName`.

Reply:
![image](https://user-images.githubusercontent.com/47894313/99674468-c0926e80-2a7e-11eb-8576-45d312139821.png)
![image](https://user-images.githubusercontent.com/47894313/99674477-c425f580-2a7e-11eb-9d84-f7032b0a3f64.png)